adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -33,10 +33,12 @@
 import android.content.pm.Signature;
 import android.os.Build;
 import android.os.Bundle;
+import android.util.Base64;
+
 import androidx.test.InstrumentationRegistry;
```

Comment:
given that we seems to be using androidx.test.platform.app.InstrumentationRegistry, can we remove this for good?

Reply:
ps. I have zero idea why there are InstrumentationRegistry in two packages.

Reply:
Yeah, missed this import - I'll run the tool

Reply:
Resolved in https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1449